app/assets/stylesheets/frontend/layouts/html-publication.scss

```diff
@@ -25,3 +25,9 @@
     padding-top: 0;
   }
 }
+
+#print-meta-data {
```

Comment:
@fofr Thanks for the heads up! 

> Why did you opt for visibility: hidden; over display: none;?
It was the preferred way to hide items in a CSS Tricks article for print version because it wasn't affecting the layout. But it makes no difference here (that I can see) so I've switched to `display:none;`

I've reimplemented this for government-frontend, where I:
- use `display:none;`
- avoid styling based on IDs
- no units added to zeros

Please see the corresponding PR on `government-frontend`: https://github.com/alphagov/government-frontend/pull/288